content/05.discussion.md

```diff
@@ -1 +1,69 @@
-## Discussion
\ No newline at end of file
+## Discussion
+ 
+Scientific news coverage, unlike academic articles, is accountable to reflecting not only the current state of research, but also the diversity of its readership.
+News coverage is the conduit between the academic and public spheres, and consequently shapes the public's view of science and scientists.
+At a minimum, news coverage should match the current academic rates of regional and gender biases.
```

Comment:
```suggestion
Though it would be best for news coverage to promote equitable representation, at a minimum news coverage should match the current rates of regional and gender participation in academia.
```